packages/itwin/property-grid/src/test/hooks/UseInstanceSelection.test.tsx

```diff
@@ -312,4 +313,72 @@ describe("useInstanceSelection", () => {
       expect(result.current.ancestorsNavigationProps.canNavigateUp).to.be.true;
     });
   });
+
+  describe("feature usage reporting", () => {
+    const initialProps: InstanceSelectionProps = {
+      imodel,
+    };
+
+    it("reports when navigates to parent", async () => {
```

Comment:
I think these tests can be combined into one, because both of them need to `navigateUp`

Reply:
Merged.